tests/test_post.py

```diff
@@ -696,6 +726,7 @@ def test_nodal_plastic_eqv_strain(mapdl, plastic_solve):
     assert np.allclose(seqv_ans, seqv_aligned)
 
 
+@skip_if_testing_minimal
```

Comment:
I think it is better to use markers that declare the features used by the test case.

That way, if you have a `minimal + one extra` installation you can test it fully

Something like `@skip_if_not_installed("graphics")` and then let declare a list of capabilities "tools-path, graphics, ..." and by default all of them are set, but for minimal tests none are set.

Reply:
Done!